src/main/kotlin/graphics/scenery/proteins/RibbonDiagram.kt

```diff
@@ -118,7 +118,9 @@ class RibbonDiagram(val protein: Protein, private val displaySS: Boolean = false
 
         val subParent = Mesh("SubProtein")
 
-        val splinePoints = spline.splinePoints().map{ it.sub(centroid) }
+        //the very first point of the spline is neglected as this makes the size of the point list divisible by the
+        //section vertices count
+        val splinePoints = spline.splinePoints().map{ it.sub(centroid) }.drop(1)
```

Comment:
Dropping one element might be a huge performance hit. Use e.g. a subList of splinePoints() instead